fairlearn/datasets/_fetch_acsincome.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+import pathlib
+
+from sklearn.datasets import fetch_openml
+from ._constants import _DOWNLOAD_DIRECTORY_NAME
+
+
+def fetch_acsincome(*, cache=True, data_home=None,
+                    as_frame=False, return_X_y=False):
+    """Load the ACS Income dataset.
+
+    Download it if necessary.
+
+    ==============   ==============
+    Samples total           1664500
+    Dimensionality               12
+    Features                   real
+    Target                     real
+    ==============   ==============
+
+    Source: Paper: Frances Ding (2021) [1]_
+
+    .. versionadded:: 0.8.0
+
+    Parameters
+    ----------
+    cache : bool, default=True
+        Whether to cache downloaded datasets using joblib.
+
+    data_home : str, default=None
+        Specify another download and cache folder for the datasets.
+        By default, all scikit-learn data is stored in '~/.fairlearn-data'
```

Comment:
```suggestion
        By default, all fairlearn data is stored in '~/.fairlearn-data'
```
This looks like a copy-paste error from when we first created one of these and that hasn't been detected. Perhaps we should fix this in all the existing _fetch_*.py files. That can totally be a separate PR if you feel like it. Otherwise let's open an issue for it.

Reply:
Thanks for noticing this! For now, I just made this its own commit on this PR